src/project.d.ts

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Bit of a drive-by here, but I was looking into something related to a discussion about project listing and ended up seeing this PR while I was noodling on something.

I think I'd lean toward inlining these types where they're used instead of breaking them out into a separate file (we also haven't yet had standalone `.d.ts` files and I'm not sure how I feel about them). These types are related in kind (describing projects) but not really in use.

Unless an interface is being used in a general way not tied to code, I think it's a little more clear to inline?

Reply:
I wasn't sure what the best way was to do this in TS or what the best practice was so thanks for chiming in! I don't have anything against this, will change to inlined. I think it does make more sense to appear with the code in this case.

Reply:
fixed in https://github.com/firebase/firebase-tools/pull/1370/commits/7bf088e24e809655043325380c6cde2388159328